content/applications/finance/fiscal_localizations/brazil.rst

```diff
@@ -0,0 +1,371 @@
+modified if needed.
+
+Configuration
+=============
+
+Modules installation
+--------------------
+
+:ref:`Install <general/install>` the following modules to get all the features of the Brazilian
+localization:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 25 50
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Brazilian - Accounting`
+     - `l10n_br`
+     - Default :ref:`fiscal localization package <fiscal_localizations/packages>` adds accounting
+       characteristics for the Brazilian localization, which represent the minimum configuration
+       required for a company to operate in Brazil. The module's installation automatically loads:
+       chart of accounts, taxes, and include required fields to properly configure the contact.
+   * - :guilabel:`Brazil - Accounting Reports`
+     - `l10n_br_reports`
+     - Includes all the technical and functional requirements to generate and validate
+       :doc:`electronic documents
+       </applications/finance/accounting/customer_invoices/electronic_invoicing>`, based on the
+       technical documentation published by the SRI. The authorized documents are: invoices, credit
+       notes, debit notes, withholdings and purchase liquidations.
+   * - :guilabel:`Avatax Brazil`
+     - `l10n_br_avatax`
+     - Add Brazilian tax calculation via Avatax and all necessary fields needed to configure Odoo in
+       order to properly use Avatax and send the needed fiscal information to retrieve the correct
+       taxes.
+   * - :guilabel:`Avatax for SOs in Brazil`
+     - `l10n_br_avatax_sale`
+     - Same as the `l10n_br_avatax` module with the extension to the sales order module.
+
+Configure your company
+----------------------
+
+To configure your company information, go to the :menuselection:`Contacts` app and search the name
+given to your company or activate :ref:`developer mode <developer-mode>` and go to
+:menuselection:`Company --> Contact` and then edit the contact.
+
+1. Select the :guilabel:`Company` option at the top of the page. Then, configure the following
+   fields:
+
+   - :guilabel:`Name`
+   - :guilabel:`Address` (add :guilabel:`City`, :guilabel:`State`, :guilabel:`Zip Code`,
+     :guilabel:`Country`)
+   - Tax ID (:guilabel:`CNPJ`)
+   - :guilabel:`IE`
+   - :guilabel:`IM`
+   - :guilabel:`SUFRAMA code` (if applicable)
+   - :guilabel:`Phone`
+   - :guilabel:`Email`
+
+.. image:: brazil/company-configuration.png
+  :align: center
+  :alt: Company configuration.
+
+2. Configure the :guilabel:`Fiscal Information` within the :guilabel:`Sales and Purchase` tab:
+
+   - Add the :guilabel:`Fiscal Position` for :ref:`Avatax Brazil <brazil/fiscal-positions>`.
+   - :guilabel:`Tax Regime`
+   - :guilabel:`ICMS Taxpayer Type`
+   - :guilabel:`Main Activity Sector`
+
+.. image:: brazil/contact-fiscal-configuration.png
+  :align: center
+  :alt: Company fiscal configuration.
+
+Finally, upload a company logo and save the contact.
+
```

Comment:
@samueljlieber can we also add an additional Note here below saying:
If you are a Simplified Regime you need to configure the ICMS rate in :menuselection:`Accounting
--> Configuration --> Settings --> Taxes --> Avatax Brazil` 

Thanks!